### DISCUSSION TITLE
Broken symbols in multiple shells and terminals

### DISCUSSION THREAD

**Discussion by joaociocca** [JanDeDobbeleer/oh-my-posh#4458] [Troubleshoot]
This is kinda weird, not sure if I should replicate all the content here or just point to https://forum.garudalinux.org/t/fonts-and-symbols/32346/7

Basically: Tokyo theme isn't showing symbols correctly. But `echo -e` for those symbols are printed as they should (kinda, they get bitten on the side).

Kitty zsh:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/6473725/eff08fe2-0bf9-42fb-b621-153d04e7aee6)

Konsole bash:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/6473725/b840dd4d-4ca8-4028-8db2-60bf6dbcf17d)

Konsole fish:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/6473725/f320ffc5-2194-4596-8b3e-812e23e0a984)

**Comment by JanDeDobbeleer** [comment]
Hmm. There's no logic inside oh-my-posh that causes this, and on Linux everything is UTF8. So, either the shell isn't configured correctly (maybe something is influencing this), or it doesn't support an extended charset somehow.

**Reply by joaociocca** [reply]
the theme file seems okay also:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/6473725/763f90dd-8017-40b9-ab98-5d2bae817d2d)

any idea where I could look for what goes wrong between theme file and prompt that messes it up?

**Reply by joaociocca** [reply]
wanna see one more crazy thing? this is before installing oh-my-posh:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/6473725/fd3616a5-0b5b-4dc7-ba75-e4fe51d0e2a8)

and this is after:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/6473725/cae5993e-8b09-4c06-9102-cc0b861f0fc1)

two main differences I note:
1- symbols are showing up correctly (yey!)
2- (but) the prompt lost it's right alignment for the second part. O.o

**Reply by joaociocca** [reply]
now THIS is bananas.

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/6473725/d9bfc986-c385-480b-a227-5bc00a2e7120)

**Reply by joaociocca** [reply]
Allacrity does the same as kitty
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/6473725/e2009fbc-240f-4019-996c-2ac7ef129578)

**Reply by JanDeDobbeleer** [reply]
Which distro is this?

**Reply by joaociocca** [reply]
Garuda dr460nized - https://garudalinux.org/downloads

**Reply by joaociocca** [reply]
Kali VM running in Virtualbox also has this issue... curious. (yes, I changed the font to the same one from those other examples)
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/6473725/fe2bbfab-cf33-479c-8b47-a3088b0cde6a)

looking more closely...

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/6473725/2ccf112f-67b6-404e-a545-b2936526a40f)

**Reply by joaociocca** [reply]
I still can't get over how bizarre this is...

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/6473725/6de100d1-bc4c-4da9-9211-25c8844c342e)

same as before, running eval again fixes the icons/emojis, but breaks the left/right alignment...
